packages/action/src/bin/github-action.js

```diff
@@ -6,6 +6,9 @@ const { main } = require("./main");
 const start = async () => {
   const args = {
     definitionFile: core.getInput("definition-file"),
+    projects: core.getInput("projects")
+      ? core.getInput("projects").split(",")
+      : undefined,
```

Comment:
```suggestion
    projects: core.getInput("projects")?.split(",")
```
I think we can add optional chaining here given that we are working with node >= 14.0